vcf2fhir/json_generator.py

```diff
@@ -30,20 +48,47 @@ def _valid_record(record):
             record,
             record.samples[0].data)
         return False
-    if(len(record.samples) == 0 or record.samples[0].gt_type is None):
+    if(
+            (len(record.samples) == 0 or record.samples[0].gt_type is None) or
+            record.samples[0]['GT'] in ['0/0', '0|0', '0'] or
+            (
+                (
+                    (
+                        (not record.is_sv or
+                            (record.is_sv and
+                                record.INFO['SVTYPE'] in [
+                                                            'INV',
+                                                            'DEL',
+                                                            'INS'
+                                                        ]))) and
+                    '.' in record.samples[0]['GT'] and
+                    genomic_source_class == 'germline'))):
         invalid_record_logger.debug(
             ("Reason: VCF FORMAT.GT is null ('./.', '.|.', '.', etc), " +
              "Record: %s, considered sample: %s"),
             record,
             record.samples[0].data)
         return False
-    if(
-            len(record.ALT) != 1 or
-            (record.ALT[0].type != 'SNV' and record.ALT[0].type != 'MNV')):
+    if not record.REF.isalpha():
+        invalid_record_logger.debug(
+            ("Reason: REF is not a simple character string. " +
+             "Record: %s, considered sample: %s"),
+            record,
+            record.samples[0].data)
+        return False
+    if not record.is_sv and not validate_alt_simple(record.ALT):
```

Comment:
Hi Shailesh, I have updated the code to use `pyvcf` for `alt validation` and I have `commented` out the code for **validation using regular expression** just in case if we need it when we test the code for structural variants. If the `alt validation` using pyvcf works fine I'll remove the code for **validation using regular expression**.